thoth/storages/sync.py

```diff
@@ -464,89 +465,82 @@ def sync_dependency_monkey_documents(
 
 def sync_inspection_documents(
     document_ids: Optional[List[str]] = None,
-    *,
     force: bool = False,
     graceful: bool = False,
     graph: Optional[GraphDatabase] = None,
-    amun_api_url: str,
-    only_ceph_sync: bool = False,
-    only_graph_sync: bool = False,
     is_local: bool = False,
 ) -> tuple:
     """Sync observations made on Amun into graph database."""
-    from amun import get_inspection_build_log
-    from amun import get_inspection_job_log
-    from amun import get_inspection_specification
-    from amun import get_inspection_status
-    from amun import has_inspection_job
-    from amun import is_inspection_finished
-
-    if is_local:
-        raise NotImplementedError("Cannot sync inspection documents from a local file")
-
-    if only_graph_sync and only_ceph_sync:
-        raise ValueError("At least one of Ceph or Graph should be performed")
-
-    inspection_store = InspectionResultsStore()
-    inspection_store.connect()
-
-    dependency_monkey_reports_store = DependencyMonkeyReportsStore()
-    dependency_monkey_reports_store.connect()
+    if is_local and not document_ids:
+        raise ValueError(
+            "Cannot sync documents from local directory without explicitly specifying a list of documents to be synced"
+        )
 
     if not graph:
         graph = GraphDatabase()
         graph.connect()
 
     processed, synced, skipped, failed = 0, 0, 0, 0
-    for inspection_id in document_ids or dependency_monkey_reports_store.iterate_inspection_ids():
-        processed += 1
-        if force or not inspection_store.document_exists(inspection_id):
-            finished = is_inspection_finished(amun_api_url, inspection_id)
+    for inspection_document_id in document_ids or InspectionStore.iter_inspections():
+        print(inspection_document_id)
+
+        if not is_local:
+            inspection_store = InspectionStore(inspection_id=inspection_document_id)
+            inspection_store.connect()
+        else:
+            main_repo = Path(f"{inspection_document_id}/results")
+        
+        for inspection_result_number in range(inspection_store.results.get_results_count()) or main_repo.iterdir():
+            print(inspection_result_number)
 
-            if finished:
-                _LOGGER.info("Obtaining results from Amun API for inspection %r", inspection_id)
+            processed += 1
+
+            if force or not graph.inspection_document_id_result_number_exists(
+                inspection_document_id=os.path.basename(inspection_document_id),
```

Comment:
Just:

```python
                inspection_document_id=inspection_document_id,
```

?

Reply:
It needs to check both actually because we enter `document_id` and `result_number` in the InspectionRun Table now, so for each inspection result there is an InspectionRun record

Reply:
The comment was addressing removal of `os.path.basename` call.

Reply:
> The comment was addressing removal of `os.path.basename` call.

sorry, I didn't understand :( thanks!